core/src/main/java/org/mule/runtime/core/internal/policy/CompositeSourcePolicy.java

```diff
@@ -180,4 +188,37 @@ public CompositeSourcePolicy(List<Policy> parameterizedPolicies,
     return concatMap;
   }
 
+  private void logPolicy(String eventId, String policyName, String message) {
+    if (LOGGER.isTraceEnabled()) {
+      //TODO Remove event id when first policy generates it. MULE-14455
+      LOGGER.trace("Event Id: " + eventId + ".\n Processing " + policyName + "\n" + message);
+    }
+  }
+
+  private String getCoreEventId(CoreEvent event) {
+    return event.getContext().getId();
+  }
+
+  private String getCoreEventAttributesAsString(CoreEvent event) {
+    return event.getMessage().getAttributes().getValue().toString();
```

Comment:
null is a valid attributes value as far as I know, so this might explode with NullPointerException